--- setup.py.orig	2019-09-30 19:16:22.578773094 +0200
+++ setup.py	2019-09-30 19:16:29.371726762 +0200
@@ -10,7 +10,6 @@
     description='Loop application synced with jack transport',
     long_description=open('readme.md').read(),
     install_requires=["JACK-Client>=0.4.5",
-                      "PyQt5>=5.11",
                       "SoundFile>=0.10",
                       "numpy>=1.16"],
     include_package_data=True,
