--- setup.py.orig	2019-12-10 21:42:51.990711367 +0100
+++ setup.py	2019-12-10 21:43:07.458798789 +0100
@@ -10,8 +10,7 @@
     author_email='superboucle@nura.eu',
     description='Loop application synced with jack transport',
     long_description=open('readme.md').read(),
-    install_requires=["PyQt5>=5.11",
-                      "SoundFile>=0.8.0",
+    install_requires=["SoundFile>=0.8.0",
                       "numpy>=1.16"],
     include_package_data=True,
     url='http://superboucle.nura.eu',
